spacy_llm/pipeline/llm.py

```diff
@@ -1,20 +1,26 @@
+import logging
 import typing
 import warnings
 from pathlib import Path
-from typing import Any, Iterable, Iterator, Optional, Tuple, Type, cast
+from typing import Any, Iterable, Iterator, List, Optional, Tuple, Type, cast
 
 import spacy
 from spacy.language import Language
 from spacy.pipeline import Pipe
 from spacy.tokens import Doc
 from spacy.vocab import Vocab
 
+
 from .. import registry  # noqa: F401
 from ..cache import Cache
 from ..compat import TypedDict
 from ..ty import LLMTask, PromptExecutor
 
 
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.NullHandler())
```

Comment:
And I'm also not sure what happens in terms of multiprocessing with this approach?

Reply:
by multiprocessing do you just mean `nlp.pipe(..., n_processes=2)`?

Reply:
Yes, with `nlp.pipe(n_process=1)` and `nlp.pipe(n_process=2)`, similar to the tests starting here: https://github.com/explosion/spaCy/blob/6930a6bf454bdae4ef12593c42896bf093315f8e/spacy/tests/test_language.py#L286

Reply:
Adding these kinds of tests separately in this PR: https://github.com/explosion/spacy-llm/pull/117. This works ok with n_process > 1 but we would need proper logging configuration in spaCy's multiprocessing stuff to make this work perfectly and to be able to follow the multiprocessing logging cookbook: https://docs.python.org/3/howto/logging-cookbook.html.

This logging to a `QueueHandler` idea would require some work on spaCy.